src/main/java/adris/altoclef/BotBehaviour.java

```diff
@@ -141,6 +141,16 @@ public void setAllowWalkThroughFlowingWater(boolean value) {
         current().applyState();
     }
 
+    //Way of disabling MLG and defense from a task
+    public boolean isDefenseDisabled() {
+        return current().disableDefence;
+    }
```

Comment:
Consider making mob defence enabled/disabled an interface. We can talk about how to integrate this later. Discord me if you are interested in learning about this,